From 0c262baaaf640a264f5b26c22771ee77a78314e3 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:24:13 -0400
Subject: [redhat] kabi: Adding symbol netlink_broadcast (net/netlink/af_netlink.c)

Message-id: <89fc4dbd3dad07ed475b6b3617a9f3c7439f5e6a.1410510773.git.poros@redhat.com>
Patchwork-id: 95963
O-Subject: [PATCH RHEL7.1 133/233] kabi: Adding symbol netlink_broadcast (net/netlink/af_netlink.c)
Bugzilla: 1077335
RH-Acked-by: Jiri Benc <jbenc@redhat.com>

Bugzilla: 1077335
    https://bugzilla.redhat.com/show_bug.cgi?id=1077335

    Description:
    ============
    Netlink API.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/netlink_broadcast
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/netlink_broadcast
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/netlink_broadcast

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/netlink_broadcast b/redhat/kabi/kabi-module/kabi_ppc64/netlink_broadcast
new file mode 100644
index 0000000..b48574a
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/netlink_broadcast
@@ -0,0 +1,2 @@
+#1-
+0xc5a58e97 netlink_broadcast vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/netlink_broadcast b/redhat/kabi/kabi-module/kabi_s390x/netlink_broadcast
new file mode 100644
index 0000000..726d557
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/netlink_broadcast
@@ -0,0 +1,2 @@
+#1-
+0xdce4e0ef netlink_broadcast vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/netlink_broadcast b/redhat/kabi/kabi-module/kabi_x86_64/netlink_broadcast
new file mode 100644
index 0000000..022cf08
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/netlink_broadcast
@@ -0,0 +1,2 @@
+#1-
+0xbd9316ae netlink_broadcast vmlinux EXPORT_SYMBOL
-- 
1.7.1